arrow-core-test/src/main/kotlin/arrow/test/laws/ComonadLaws.kt

```diff
@@ -22,9 +21,7 @@ object ComonadLaws {
         Law("Comonad Laws: duplicate then map into extract is identity") { CM.duplicateThenMapExtractIsId(GEN, EQ) },
         Law("Comonad Laws: map and coflatMap are coherent") { CM.mapAndCoflatmapCoherence(GEN, EQ) },
         Law("Comonad Laws: left identity") { CM.comonadLeftIdentity(GEN, EQ) },
-        Law("Comonad Laws: right identity") { CM.comonadRightIdentity(GEN, EQ) },
-        Law("Comonad Laws: cokleisli left identity") { CM.cokleisliLeftIdentity(GEN, EQ) },
-        Law("Comonad Laws: cokleisli right identity") { CM.cokleisliRightIdentity(GEN, EQ) }
```

Comment:
These are alternative definitions only added for parity with monad laws. They follow directly from the other laws and are thus not needed